pages/protect-your-data.jsx

```diff
@@ -0,0 +1,132 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var CCLicenseNote = require('../components/cc-license-note.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Protect Your Data</h1>
+        <section className="intro">
+          <Illustration
+          height={204} width={204}
+          src1x="/img/pages/protect-your-data/photo-protect-your-data.png"
+          src2x="/img/pages/protect-your-data/photo-protect-your-data@2x.png"
+          alt=""
+          className="img-circle">
```

Comment:
It's kinda weird to me that `Illustration` generates a link with no `href` if `link` isn't passed in as a prop, but that's probably out of the scope of this PR.